### PR TITLE
JS runtime: Extract `RESUME` function

### PR DESCRIPTION
**Pull request by dvdvgt**

### CODE REVIEW

File: libraries/js/effekt_runtime.js [hunk 1]
```diff
@@ -126,9 +126,24 @@ function SHIFT(p, body, ks, k) {
     return () => c(meta, k2)
   }
 
-  let k1 = meta.stack
+  const k1 = meta.stack
   meta.stack = null
-  return body(resumeComp, meta, k1)
+  const resume = (c, ks, k) => RESUME(c, cont, ks, k)
+  return body(resume, meta, k1)
```
Comment (b-studios) [inline libraries/js/effekt_runtime.js:132]: Do you plan to change this to: 

```
body(c, meta, k1)
```

?

Comment (dvdvgt) [reply libraries/js/effekt_runtime.js:132]: No, I think it would not work if I did so.

**Comment by b-studios** [comment]
I took the liberty to make the above mentioned changes.

**Comment by dvdvgt** [comment]
Was just about to push the cherry-picked commits myself. Thanks for the help!

**Comment by b-studios** [comment]
Sorry if this duplicated work that you wanted to do :(

**Comment by b-studios** [comment]
From my side, we can merge this now. Thanks for your help -- finally I can put #679 to rest :)

**Comment by dvdvgt** [comment]
Not sure whether I was of much help here, but sure :)

**Pull request merged**